actions/class.Items.php

```diff
@@ -153,6 +153,15 @@ public function getOntologyData()
         return parent::getOntologyData();
     }
 
+    /**
+     * view class label action
+     * @requiresRight classUri READ
+     */
+    public function viewClassLabel()
```

Comment:
I think we do not need this here, instead we should use annotations and create proper method: 

https://github.com/oat-sa/tao-core/pull/2852/files#r612168872